### PR TITLE
Should be ready to merge into feature now

### PR DESCRIPTION
**Pull request by cyberW0MBAT**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,43 +21,91 @@
...
+		  <span>K</span>
+		  <span>o</span>
+		  <span>z</span>
+		  <span>a</span>
+		  <span>r</span>
+		  <span>e</span>
+		  <span>v</span>
+		</div>
+		<h1>Hello, I'm Genadi Kozarev.</h1>
+		<h1>I'm a full-stack software developer.</h1>
```
Comment (Albertoroldanq) [inline index.html:42]: I am not sure if having two `<h1>` is good practice

File: style.css [hunk 2]
```diff
@@ -8,7 +8,15 @@ html {
...
-	background-color: #00BBF9;
+  background-color: #00BBF9;
+}
+
+nav {
+  background-color: #00F5D4;
+}
+
+h1 {
+	font-size: 3rem;
```
Comment (Albertoroldanq) [inline style.css:19]: indentation

File: style.css [hunk 3]
```diff
@@ -57,68 +67,131 @@ nav {
...
+  scroll-margin-top: 5rem;
+}
+
+body { background: @bg;}
+a#arrowEffect { color: white; text-decoration: none; }
+
+.arrow {
+  text-align: center;
+  margin: 8% 0;
+}
```
Comment (Albertoroldanq) [inline style.css:112]: missing one line of space

File: style.css [hunk 4]
```diff
@@ -57,68 +67,131 @@ nav {
...
+  color: #00F5D4;
+  box-shadow: 0 0 11px rgba(33,33,33,.2); 
+}
+
+#anchorToPortfolio {
+  scroll-margin-top: 5rem;
+}
+
+body { background: @bg;}
+a#arrowEffect { color: white; text-decoration: none; }
```
Comment (Albertoroldanq) [inline style.css:107]: maybe add the properties of this selectors in a new line?

File: index.html (hunk 1 above)
Comment (cyberW0MBAT) [reply index.html:42]: I didn't find a problem with that on the internet so I'll take my chances with the trainer :D

File: style.css (hunk 2 above)
Comment (cyberW0MBAT) [reply style.css:19]: Yeah sometimes TextMate makes the "tab" indentation weird but I'll fix it now. Thanks

File: style.css (hunk 3 above)
Comment (cyberW0MBAT) [reply style.css:112]: Absolute eagle eye ! Thanks :D

File: style.css [hunk 5]
```diff
@@ -57,68 +67,131 @@ nav {
...
   cursor: pointer;
+  font-size: 2rem;
+}
+
+a.portfolioButton:hover {
+  color: #00F5D4;
+  box-shadow: 0 0 11px rgba(33,33,33,.2); 
+}
+
+#anchorToPortfolio {
```
Comment (Albertoroldanq) [inline style.css:102]: it's alright using ID as selector, but I think is better practice to use a class instead

File: style.css (hunk 4 above)
Comment (cyberW0MBAT) [reply style.css:107]: done

File: style.css (hunk 5 above)
Comment (cyberW0MBAT) [reply style.css:102]: Agree. It was just easier than creating a new class only for this tag. Thanks

File: style.css [hunk 6]
```diff
@@ -57,68 +67,131 @@ nav {
...
+  margin-left: 1rem;
 }
 
 .descriptionAboutMe, p.whiteText {
-	font-size: 1.3rem;
-	margin: 1rem;
-	text-align: justify;
+  font-size: 1.3rem;
+  margin: 1rem;
+  text-align: justify;
```
Comment (Albertoroldanq) [inline style.css:186]: need some indentation

**Review by Albertoroldanq — CHANGES_REQUESTED** [review]
that's quite an improvement from the last review man! Just a couple of comments , but your website looks nice! Well done with the animation effects, they make your site look fancy

**Pull request merged**